### PR TITLE
gen publisher token with server signKey

### PR DESCRIPTION
**Pull request by hunshenshi**

### CODE REVIEW

File: pkg/models/project.go [hunk 1]
```diff
@@ -33,7 +33,4 @@ type ProjectBase struct {
 	Version string         `db:"f_version,default=''" json:"version,omitempty"`  // Version project version
 	Proto   enums.Protocol `db:"f_proto,default='0'"  json:"protocol,omitempty"` // Proto project protocol for event publisher
 	Description string         `db:"f_description,default=''"    json:"description,omitempty"`
```
Comment (saitofun) [inline pkg/models/project.go:35]: code fmt

File: pkg/modules/publisher/publisher.go [hunk 2]
```diff
@@ -198,17 +193,13 @@ func RemovePublisher(ctx context.Context, r *RemovePublisherReq) error {
...
 	_, l = l.Start(ctx, "UpdatePublisher")
 	defer l.End()
 
-	publisherJwt := &jwt.Jwt{
-		Issuer:  project.ProjectBase.Issuer,
-		ExpIn:   project.ProjectBase.ExpIn,
-		SignKey: project.ProjectBase.SignKey,
-	}
-	token, err := publisherJwt.GenerateTokenByPayload(publisherID)
+	token, err := publisherJwt.GenerateTokenWithoutExpByPayload(publisherID)
```
Comment (saitofun) [inline pkg/modules/publisher/publisher.go:202]: Token don't need updating, publisherID is not changed

File: pkg/depends/conf/jwt/jwt.go [hunk 3]
```diff
@@ -37,6 +37,18 @@ func (c *Jwt) GenerateTokenByPayload(payload interface{}) (string, error) {
 	return token.SignedString([]byte(c.SignKey))
 }
 
+func (c *Jwt) GenerateTokenWithoutExpByPayload(payload interface{}) (string, error) {
```
Comment (Liuhaai) [inline pkg/depends/conf/jwt/jwt.go:40]: a new func is unnecessary

File: pkg/modules/event/event.go [hunk 4]
```diff
@@ -61,40 +61,26 @@ func OnEventReceived(ctx context.Context, projectName string, r *eventpb.Event)
...
-		if len(r.Header.EventType) > 0 {
-			eventType = r.Header.EventType
-		}
-		if len(r.Header.Token) > 0 {
-			if err = publisherVerification(ctx, r, l); err != nil {
-				l.Error(err)
-				return
-			}
-		}
+	eventType, err = checkHeader(ctx, projectName, l, ret, r)
```
Comment (huangzhiran) [inline pkg/modules/event/event.go:64]: check func only return err for judge

Comment (hunshenshi) [reply pkg/modules/event/event.go:64]: if you don't need it, i will move it

**Review by saitofun — APPROVED** [review]

**Pull request merged**